src/main/java/uk/gov/hmcts/reform/sscs/ccd/presubmit/postpone/PostponeHearingHandler.java

```diff
@@ -27,10 +27,6 @@ public boolean canHandle(CallbackType callbackType, Callback<SscsCaseData> callb
 
     @Override
     public PreSubmitCallbackResponse<SscsCaseData> handle(CallbackType callbackType, Callback<SscsCaseData> callback, String userAuthorisation) {
-        if (!canHandle(callbackType, callback)) {
```

Comment:
Is this not needed? My recollection is we added this to all handlers as a safety net to stop handlers incorrectly getting called.

Reply:
is it not called in the PreSubmitCallbackDispatcher so will always be called anyway before the handle method is called?

Reply:
ok